README.md

```diff
@@ -107,13 +107,16 @@ npm test
 
 ### Functional tests
 
-Update `.func_config.json` with your own username, github token, access key, and host:
+Fork `functional-*` repositories to your organization from [screwdriver-cd-test](https://github.com/screwdriver-cd-test)
```

Comment:
I was really hoping that the [Toolbox](https://github.com/screwdriver-cd/toolbox) repository would have something that would configure a test org for you to run your functional tests. This is just a nice-to-have.

What you have here is good 👍 . I believe your additional documentation covers the manual steps well.